<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>258</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>258</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>258</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TBMP (by Archangel)</string>
  </property>
  <widget class="QPushButton" name="btn_play">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>140</y>
     <width>88</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_exit">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>220</y>
     <width>88</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_stop">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>220</y>
     <width>88</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_choose">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>88</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_nowplay">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>8</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Now playing</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="path_wave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QSlider" name="sld_volume">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>20</width>
     <height>160</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_pause">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>140</y>
     <width>88</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_addmusic">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_dropmusic">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Drop</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_volume_value">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>31</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="txted_playlist">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>291</width>
     <height>84</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_nothing_to_play">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>177</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(197, 0, 0);
color: rgb(255, 255, 255);
font: 75 italic 20pt &quot;Noto Sans&quot;;</string>
   </property>
   <property name="text">
    <string>Nothing to Play</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>124</x>
     <y>120</y>
     <width>271</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_previous">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>140</y>
     <width>41</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_next">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>140</y>
     <width>41</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_current_index">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>16</y>
     <width>21</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_loop">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>220</y>
     <width>41</width>
     <height>34</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Loop</string>
   </property>
  </widget>
  <zorder>btn_pause</zorder>
  <zorder>btn_stop</zorder>
  <zorder>btn_play</zorder>
  <zorder>btn_exit</zorder>
  <zorder>btn_choose</zorder>
  <zorder>lbl_nowplay</zorder>
  <zorder>path_wave</zorder>
  <zorder>sld_volume</zorder>
  <zorder>btn_addmusic</zorder>
  <zorder>btn_dropmusic</zorder>
  <zorder>lbl_volume_value</zorder>
  <zorder>txted_playlist</zorder>
  <zorder>lbl_nothing_to_play</zorder>
  <zorder>progressBar</zorder>
  <zorder>btn_previous</zorder>
  <zorder>btn_next</zorder>
  <zorder>lbl_current_index</zorder>
  <zorder>btn_loop</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
